Gemfile.lock

```diff
@@ -75,7 +75,7 @@ GEM
       zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    annotate (3.0.3)
+    annotate (3.1.1)
```

Comment:
why were gems updated?

Reply:
I actually ran `bundle update` while trying to diagnose why circleci was failing to run the `code_analysis` task (spoiler, it wasn't related). Left it updated because I thought it would do no harm. I can roll this update back if you think this should be done in another PR or not done at all right now.

Reply:
I usually prefer having PRs tackle a specific thing and not doing more than that, but I can make an exception since I agree it does no harm 😄